src/ert/logging/logger.conf

```diff
@@ -87,6 +85,14 @@ loggers:
   res.config:
     level: WARNING
     handlers: [stderr, file]
+  shapely:
+    level: INFO
+  subscript:
+    level: INFO
```

Comment:
- How did you choose the packages to be included here (h5py, shapely and subscript)?  Was it based on the logs from Drogon?
- Is there a default option if we don't specify the level for a given package?

Reply:
These package names are taken from an observation of what fills up the terminal window (or the ert log files) when running Drogon or similar. 

(the default is DEBUG, which is slightly unfortunate for some external packages that might be very chatty at this level, notably `subscript.fmuobs.parsers` in the case of Drogon.)